contracts/Trustcoin.sol

```diff
@@ -0,0 +1,112 @@
+pragma solidity ^0.4.8;
+
+import './lib/ERC20.sol';
+import './lib/SafeMath.sol';
+
+contract Trustcoin is ERC20, SafeMath {
+
+  string public name = 'Trustcoin';
+  uint8 public decimals = 18;
+  string public symbol = 'TRST';
+  string public version = 'TRST1.0';
+  uint256 public totalSupply = 100000000; // One hundred million
+  uint256 public totalMigrated;
+  address public newToken;
+
+  mapping(address => uint) balances;
+  mapping (address => mapping (address => uint)) allowed;
+
+  bool migrationActive;
```

Comment:
for extra security, please explicitly state for each variable and function whether they're internal, public or external.